changelog_entry.yaml

```
@@ -0,0 +1,7 @@
+- bump: patch
+  changes:
+    removed:
+      - "status and message values from build_metadata function"
+    changed:
+      - "metadata_route to add status and message values to the
```

```suggestion
      - metadata_route to add status and message values to the response object
```

__nitpick, nonblocking:__ Remove quotes and ensure items are on one line

While YAML does allow for items to be spread across lines, we don't typically do so, and tend to add lines without quotes